inst/schema/ComparisonBarchartMetadata.schema.json

```diff
@@ -10,11 +10,12 @@
       "type": "array",
       "items": { "$ref": "Filter.schema.json" }
     },
-    "defaults": {
+    "defaults": { "$ref": "BarchartDefaults.schema.json" },
```

Comment:
OK, this seems a bit weird to me, but it would certainly include all the metadata you need to set the default. But doesn't this mean you're sending the full default selections for the default indicator twice, because they'll be in the `selections` array as well? Why not just send `defaultIndicator` as a string, and let HINT pick out the corresponding selections from the array (or use the first in the list if not provided? Don't feel particularly strongly about this as amount of data is low, but surprised at the repetition implied here.

Reply:
Oh, actually, is the reason so that HINT can just keep using `defaults` in the same way as it does now, directly from the response, with no code changes at all? And then to change indicators, it could actually edit the response object in place by swapping in the newly selected indicator's selections? That would be even less code to change than my 'facade' suggestion. I never really like editing the response object (can make things harder to debug), but that would certainly mean the change in HINT would be minimal.

Reply:
That sounds good to me, agree funny to dupe data but it isn't a huge amount. I think in mind of time lets go with this for now as Lekan has his branch working with it. Let's make sure we review when we come to doing the full filtering of filters/default selections for all plots PR next year